app/views/practices/show/implementation/_resource_file_attachment.html.erb

```diff
@@ -3,8 +3,12 @@
   <% file_attachments.each do |fa| %>
     <div class="margin-bottom-1">
       <% if fa.attachment.present? %>
+        <%
+          fa.name.length == 0 ? s_name = fa.attachment_file_name : s_name = fa.name
```

Comment:
if we do `.length` on something that doesn't exist we are going to error out... so maybe we can do 
```
s_name = fa.name ? fa.name : fa.attachment_file_name
s_description = fa.description ? fa.description : fa.attachment_file_name
```

Reply:
we can shorten even more to
```
 s_name = fa.name || fa.attachment_file_name
s_description = fa.description || fa.attachment_file_name
```
i just realized hahah

Reply:
Hey Elisa  I see what you are trying to prevent - but neither of those work.. bc the name and description fields are non - nullable string fields (char varying)...  which doesn't require a length... so in both cases above... s_name would be set to fa.name which could be an empty string (not null).  i.e.,
![image](https://user-images.githubusercontent.com/60527788/105217106-4dea5180-5b08-11eb-9467-8eb859a84ddc.png)

Reply:
oo gotcha then could we do...
```
s_name = fa.name.blank? ? fa.attachment_file_name : fa.name
s_description = fa.description.blank? ? fa.attachment_file_name : fa.description
```

Reply:
Yeah - that works as well.  pushed those changes.